core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,11 +96,15 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
                 info(this, s"Completion counts: [${invokerCounts.mkString(", ")}]")
                 info(this, s"Invoker health: [${health.mkString(", ")}]")
             }
-            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth()) ++
-                getUserActivationCounts()
+            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth())
         })
 
     /**
+     * A bridge method to impedance match against RestAPI.
+     */
+    def acceptRequest(lbr: LoadBalancerReq) = publish(lbr._1)(lbr._2)
```

Comment:
Yep I agree. I can contribute a commit to this PR if you want, but I feel like we should get rid of these bridge methods, if they are of no real help. And we're leaking `loadBalancer` into Entitlement anyway